BibBuddy/app/src/main/java/de/bibbuddy/SearchFragment.java

```diff
@@ -1,25 +1,66 @@
 package de.bibbuddy;
 
 import android.os.Bundle;
+import android.view.KeyEvent;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.Toast;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import com.google.android.material.textfield.TextInputLayout;
 
 /**
  * The SearchFragment is responsible for the search of books.
  *
- * @author Claudia Schönherr
+ * @author Claudia Schönherr, Luis Moßburger
  */
 public class SearchFragment extends Fragment {
 
+  private TextInputLayout searchField;
+  private EditText searchFieldText;
+  private Thread thread;
+  private IsbnRetriever isbnRetriever;
+
   @Nullable
   @Override
   public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                            @Nullable Bundle savedInstanceState) {
     View view = inflater.inflate(R.layout.fragment_search, container, false);
+    searchField = view.findViewById(R.id.searchField);
+    searchFieldText = searchField.getEditText();
+    searchFieldText.setOnKeyListener(new View.OnKeyListener() {
+      public boolean onKey(View v, int keyCode, KeyEvent event) {
+        handleIsbnInput(keyCode, event);
+        return true;
+      }
+    });
```

Comment:
Newline for better readability.
```suggestion
    });
    
```